docs/tutorials/build/sync-table.md

```diff
@@ -0,0 +1,982 @@
+
+    const BookSchema = coda.makeObjectSchema({
+
+    });
+
+    pack.addSyncTable({
+      name: "Books",
+      description: "Lists books in the collection.",
+      identityName: "Book",
+      schema: BookSchema,
+      // ...
+    });
+    ```
+
+    </div>
+    </section>
+
+=== ":material-numeric-2-circle: Define title property"
+
+    <section class="tutorial-row" markdown>
+    <div markdown>
+
+    An object schema primarily contains property definitions, which describe the different pieces of data being stored in the schema. Create a `properties` object to hold these properties.
+
+    One piece of data you'll want to store for each row is the title of the book. To create that property, add a new key-value pair to the `properties` object. The key (left side) determines two things:
+
+    1. How users will access the property's value in the doc.
+    1. Where Coda can find that value in the rows your sync formula returned.
+
+    Your sync formula is currently returning the API data as-is, and it returns the title under the key `title`. That's also a great name to expose to our users, so use it as the key of the property.
+
+    The value (right side) is another schema definition, describing what type of data that property will hold. Create a simple schema with the type `String`, since the title is a text value.
+
+    </div>
+    <div markdown>
+
+    ```{.ts hl_lines="2-4"}
+    const BookSchema = coda.makeObjectSchema({
+      properties: {
+        title: { type: coda.ValueType.String },
+      },
+    });
+    ```
+
+    </div>
+    </section>
+
+=== ":material-numeric-3-circle: Define book ID property"
+
+    <section class="tutorial-row" markdown>
+    <div markdown>
+
+    In order to function correctly a sync table's schema must include at least one property which is a unique identifier for that row. A book's title isn't guaranteed to be unique, but luckily the API response includes an `id` field you can use.
+
+    It's a best practice to avoid creating a property named exactly `id`, but instead use the pattern `{thing}Id`, or `bookId` in this case. When the name of the property differs from the key in the API response, use the `fromKey` field of the schema to connect them.
+
+    </div>
+    <div markdown>
+
+    ```{.ts hl_lines="4-7"}
+    const BookSchema = coda.makeObjectSchema({
+      properties: {
+        title: { type: coda.ValueType.String },
+        bookId: {
+          type: coda.ValueType.Number,
+          fromKey: "id",
+        },
+      },
+    });
+    ```
+
+    </div>
+    </section>
+
+=== ":material-numeric-4-circle: Configure schema settings"
+
+    <section class="tutorial-row" markdown>
+    <div markdown>
+
+    In addition the properties themselves, the schema contains various settings.
+
+    The `displayProperty` setting is used to specify which of the properties you defined should be shown within the chip in that row. For this schema the `title` property would make a good display value.
```

Comment:
Nit, optional, "within the chip in that row or wherever this row is referenced from elsewhere in the doc", something like that.

Reply:
Tried another wording, but I think captures the gist?